Binary files rkbin.orig/.git/index and rkbin/.git/index differ
diff -rupN rkbin.orig/RKBOOT/RK3566MINIALL.ini rkbin/RKBOOT/RK3566MINIALL.ini
--- rkbin.orig/RKBOOT/RK3566MINIALL.ini	2023-05-25 02:47:56.775668998 +0000
+++ rkbin/RKBOOT/RK3566MINIALL.ini	2023-05-25 02:50:49.677278682 +0000
@@ -18,8 +18,3 @@ FlashData=bin/rk35/rk3566_ddr_1056MHz_v1
 FlashBoot=bin/rk35/rk356x_spl_v1.12.bin
 [OUTPUT]
 PATH=rk356x_spl_loader_v1.13.112.bin
-[SYSTEM]
-NEWIDB=true
-[FLAG]
-471_RC4_OFF=true
-RC4_OFF=true
diff -rupN rkbin.orig/RKBOOT/RK3566MINIALL_NAND.ini rkbin/RKBOOT/RK3566MINIALL_NAND.ini
--- rkbin.orig/RKBOOT/RK3566MINIALL_NAND.ini	2023-05-25 02:47:56.775668998 +0000
+++ rkbin/RKBOOT/RK3566MINIALL_NAND.ini	2023-05-25 02:50:14.800147306 +0000
@@ -18,8 +18,3 @@ FlashData=bin/rk35/rk3566_ddr_1056MHz_v1
 FlashBoot=bin/rk35/rk356x_spl_nand_v1.07.bin
 [OUTPUT]
 PATH=rk3566_spl_loader_nand_v1.11.107.bin
-[SYSTEM]
-NEWIDB=true
-[FLAG]
-471_RC4_OFF=true
-RC4_OFF=true
diff -rupN rkbin.orig/RKBOOT/RK3566MINIALL_ULTRA.ini rkbin/RKBOOT/RK3566MINIALL_ULTRA.ini
--- rkbin.orig/RKBOOT/RK3566MINIALL_ULTRA.ini	2023-05-25 02:47:56.775668998 +0000
+++ rkbin/RKBOOT/RK3566MINIALL_ULTRA.ini	2023-05-25 02:51:21.658316038 +0000
@@ -18,8 +18,3 @@ FlashData=bin/rk35/rk3566_ddr_1056MHz_ul
 FlashBoot=bin/rk35/rk356x_spl_v1.12.bin
 [OUTPUT]
 PATH=rk356x_spl_loader_v1.09.112.bin
-[SYSTEM]
-NEWIDB=true
-[FLAG]
-471_RC4_OFF=true
-RC4_OFF=true
diff -rupN rkbin.orig/RKBOOT/RK3568MINIALL.ini rkbin/RKBOOT/RK3568MINIALL.ini
--- rkbin.orig/RKBOOT/RK3568MINIALL.ini	2023-05-25 02:47:56.775668998 +0000
+++ rkbin/RKBOOT/RK3568MINIALL.ini	2023-05-25 02:51:46.503072883 +0000
@@ -18,8 +18,3 @@ FlashData=bin/rk35/rk3568_ddr_1560MHz_v1
 FlashBoot=bin/rk35/rk356x_spl_v1.12.bin
 [OUTPUT]
 PATH=rk356x_spl_loader_v1.13.112.bin
-[SYSTEM]
-NEWIDB=true
-[FLAG]
-471_RC4_OFF=true
-RC4_OFF=true
diff -rupN rkbin.orig/RKBOOT/RK3568MINIALL_NAND.ini rkbin/RKBOOT/RK3568MINIALL_NAND.ini
--- rkbin.orig/RKBOOT/RK3568MINIALL_NAND.ini	2023-05-25 02:47:56.775668998 +0000
+++ rkbin/RKBOOT/RK3568MINIALL_NAND.ini	2023-05-25 02:52:07.383568324 +0000
@@ -18,8 +18,3 @@ FlashData=bin/rk35/rk3568_ddr_1056MHz_v1
 FlashBoot=bin/rk35/rk356x_spl_nand_v1.07.bin
 [OUTPUT]
 PATH=rk356x_spl_loader_nand_v1.11.107.bin
-[SYSTEM]
-NEWIDB=true
-[FLAG]
-471_RC4_OFF=true
-RC4_OFF=true
diff -rupN rkbin.orig/RKBOOT/RK3568MINIALL_RAMBOOT.ini rkbin/RKBOOT/RK3568MINIALL_RAMBOOT.ini
--- rkbin.orig/RKBOOT/RK3568MINIALL_RAMBOOT.ini	2023-05-25 02:47:56.775668998 +0000
+++ rkbin/RKBOOT/RK3568MINIALL_RAMBOOT.ini	2023-05-25 02:52:15.583764665 +0000
@@ -18,8 +18,3 @@ FlashData=bin/rk35/rk3568_ramboot_null0.
 FlashBoot=bin/rk35/rk3568_ramboot_null1.bin
 [OUTPUT]
 PATH=rk356x_ramboot_loader_v1.11.108.bin
-[SYSTEM]
-NEWIDB=true
-[FLAG]
-471_RC4_OFF=true
-RC4_OFF=true
diff -rupN rkbin.orig/RKBOOT/RK3568MINIALL_SPI_NAND.ini rkbin/RKBOOT/RK3568MINIALL_SPI_NAND.ini
--- rkbin.orig/RKBOOT/RK3568MINIALL_SPI_NAND.ini	2023-05-25 02:47:56.775668998 +0000
+++ rkbin/RKBOOT/RK3568MINIALL_SPI_NAND.ini	2023-05-25 02:52:27.772058287 +0000
@@ -18,8 +18,3 @@ FlashData=bin/rk35/rk3568_ddr_1332MHz_v1
 FlashBoot=bin/rk35/rk3568_miniloader_spinand_v1.13.bin
 [OUTPUT]
 PATH=rk3568_loader_spinand_v1.11.112.bin
-[SYSTEM]
-NEWIDB=true
-[FLAG]
-471_RC4_OFF=true
-RC4_OFF=true
